src/StoreProvider.spec.lua

```diff
@@ -27,4 +28,34 @@ return function()
 			Roact.mount(element)
 		end).to.throw()
 	end)
+
+	it("should accept multiple children", function()
+		local store = Rodux.Store.new(function()
+			return 0
+		end)
+
+		local element = Roact.createElement(StoreProvider, {
+			store = store
+		}, {
+			test1 = Roact.createElement("Frame"),
+			test2 = Roact.createElement("Frame"),
+			test3 = Roact.createElement("Frame")
+		})
+
+		expect(element).to.be.ok()
+
+		local handle = Roact.mount(element, folder, "StoreProvider-test")
+
+		local children = folder:GetChildren()
+		local count = 0
+
+		for i, v in ipairs(children) do
+			count  = count + 1
+		end
+
+		expect(count).to.be.equal(3)
```

Comment:
Instead of just counting here, we should assert about each individual child. I would write something like:

```lua
expect(folder:FindFirstChild("test1")).to.be.ok()
```

Additionally, for counting, you should be able to use `#children` instead of writing a loop over the children.